include/devicetree.h

```diff
@@ -464,6 +464,48 @@
  */
 #define DT_NODE_FULL_NAME(node_id) DT_CAT(node_id, _FULL_NAME)
 
+/**
```

Comment:
Nothing wrong, just a minor observation on the commit message, and one cannot place a comment in the commit message directly.

Commit c3f6f6cd7c9f772512c1e4c3ee90400014bdc9f0 says:
```
The awkward naming is due to `DT_NODELABEL` already being defined to get a node from a
nodelabel.
```

Which awkward naming ?
When you refer to an awkward naming in the commit message, then it would be nice if you also presented that particular name in the same message, so reader won't have to look at the actual changes.

Maybe like:
```
Adds the functions DT_NODE_NUM_NODELABEL and DT_NODE_NODELABEL_BY_IDX
to query the node labels of a node. The awkward naming is due to `DT_NODELABEL`
already being defined to get a node from a nodelabel.
```